app/models/grda_warehouse/project_access_group_member.rb

```diff
@@ -0,0 +1,13 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# DB view
```

Comment:
@ttoomey can you add a bit more description here about what this view is doing, and maybe a link to the view migration? (Though I realize that might go away eventually)